test/integrations/destinations/zapier/processor/data.ts

```diff
@@ -703,4 +759,93 @@ export const data = [
       },
     },
   },
+  {
+    name: 'zapier',
+    description: 'Test 5',
+    feature: 'processor',
```

Comment:
test 5 should be id and description should be something around what scenario test case is describing or summary of test case.
include scenario as well here